File: workspace/pset6/mario.py
n = -1
while n < 0 or n > 23:
    n = int(input("Enter a positive integer no greater than 23\n"))



for i in range(n):

    x = n - i

    for j in range (x):
        print(" ", end="")

    for j in range(i):
        print("#",end="")

    print ("  ",end="")

    for j in range(i):
        print("#",end="")


    print()

File: workspace/pset1/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int y;
    int x;
    int spaces;
    int hashes;

    do 
    {
        printf("Input positive number no greater than 23\n");
    
        y = get_int();

    
    }
    while (y < 0 || y > 23);



    for (x = 0; x < y; x++ ) 
        {

        for (spaces = x; spaces<y-1;spaces++) 
        {
            printf(" ");
        }

        for (hashes = 0; hashes<x+2;hashes++) 
        {
            printf("#");
    
        }

        printf("\n");
    }



}

